### PR TITLE
Replace webpack with parcel

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

**Comment by ry** [comment]
What's the right way to create a temp dir in windows?

**Comment by piscisaureus** [comment]
* `tmproot = process.platform === "win32" ? process.env.TEMP : "/tmp"`
* add a script to just run parcel without running build
* Are you going to remove webpack from deps?

File: package.json [hunk 1]
```diff
@@ -21,10 +21,10 @@
     "acorn": "^5.3.0",
     "codemirror": "^5.33.0",
     "d3": "^4.12.2",
-    "he": "^1.1.1",
```
Comment (piscisaureus) [inline package.json]: The `he` dependency is required for gendoc.

Comment (ry) [reply package.json]: No longer, now that react generates the html.

**Comment by ry** [comment]
webpack is removed from deps in last commit

**Pull request merged**